store/postgres/org.go

```diff
@@ -26,6 +27,9 @@ const (
 	createOrganizationQuery = `INSERT INTO organizations(name, slug, metadata) values($1, $2, $3) RETURNING id, name, slug, metadata, created_at, updated_at;`
 	listOrganizationsQuery  = `SELECT id, name, slug, metadata, created_at, updated_at from organizations;`
 	updateOrganizationQuery = `UPDATE organizations set name = $2, slug = $3, metadata = $4, updated_at = now() where id = $1 RETURNING id, name, slug, metadata, created_at, updated_at;`
+	addOrganizationAdmins   = `INSERT INTO relations(subject_namespace_id, subject_id, object_namespace_id, object_id, role_id) values %s RETURNING subject_id;`
+	listOrganizationAdmins  = `SELECT subject_id from relations where object_id = $1 and role_id = 'organization_admin';`
+	removeOrganizationAdmin = `DELETE from relations where object_id = $1 and subject_id = $2 and role_id = 'organization_admin';`
 )
```

Comment:
@krtkvrm @rsbh Are `subject_id`,  `object_id` and `role_id` enough to form a composite candidate key? Or can there be a case for duplication? 

P.S. Using in `listOrganizationAdmins` and `removeOrganizationAdmin` queries.